tests/test_iteration.py

```diff
@@ -0,0 +1,118 @@
+from __future__ import division
+from math import ceil
+from operator import mul
+from functools import reduce
+from itertools import product
+
+import pytest
+from nchelpers.iteration import chunk_corners, chunk_slices, chunks, \
+    opt_chunk_shape
+
+import numpy as np
+
+
+def size(shape):
```

Comment:
Numpy arrays already have a [`size` attribute](https://docs.scipy.org/doc/numpy-1.14.0/reference/generated/numpy.ndarray.size.html) that gives you what you want.

Reply:
Yes, but this is applied to other things than (just) numpy arrays. In particular chunk shapes.